src/services/storageService.ts

```diff
@@ -66,3 +71,26 @@ export const getUsers = async (): Promise<Map<string, IUser>> => {
     })
     .catch((error: Error) => Promise.reject(new Error(`Unable to retrieve value for key: ${STORAGE_USERS_KEY}. ${error.message}`)));
 };
+
+export const deleteUser = async (userId: string): Promise<void> => {
+  debug(`deleteUser(${userId})...`);
+
+  await AsyncStorage.getItem(STORAGE_USERS_KEY)
+    .then((result: string | null) => {
+      if (!result || result === '{}') {
+        debug(`deleteUser(${userId}) no users found`);
+        return;
+      }
+
+      const users: Map<string, IUser> = new Map<string, IUser>();
+      JSON.parse(result).forEach(([key, value]: [string, IUser]) => {
+        users.set(key, value);
+      });
+
+      debug(`deleteUser(${userId}) deleting user`);
+      users.delete(userId);
+
+      AsyncStorage.setItem(STORAGE_USERS_KEY, JSON.stringify(Array.from(users)));
+    })
+    .catch((error: Error) => Promise.reject(new Error(`Unable to set value for key: ${STORAGE_PIN_KEY}. ${error.message}`)));
```

Comment:
translate missing

Reply:
we are not showing this to a user, and we are not translating any other error. do you want me to add this?